### ISSUE TITLE
My Goals - WCAG compliance

### ISSUE THREAD

**Issue by steffipravasi** [bcgov/performance#1018]

**Comment by jessicahjwu** [comment]
Pipelines: To be estimated
No sprints nor epics assigned
Estimate: 2

**Comment by steffipravasi** [comment]
@jizhaogit June 20 -  

6,8,11,12, 17 
Let me know if any issues needs to be separated

**Comment by jizhaogit** [comment]
@steffipravasi Hi Steffi, would you please help separate 6,11,12, & 17 to other tickets? let's take care of these later when we have a good solution. thank you!

**Comment by steffipravasi** [comment]
@jizhaogit Tickets have been separated

**Comment by jizhaogit** [comment]
Than you @steffipravasi , #8 should be addressed

**Comment by steffipravasi** [comment]
@jizhaogit June 25 – Issue 8 points ii and iii not fixed

**Comment by steffipravasi** [comment]
The issue 8 is separated in ticket 1324. Moving this ticket now.
